yt_idefix/data_structures.py

```diff
@@ -662,11 +662,13 @@ def _get_input_parameter(self, match: re.Match) -> str:
         """Replace matched input parameters with its value"""
         key = match.group(1)
         if key not in self.parameters.get("Parameters", {}):
-            if not self._inifile:
+            if not os.path.exists(self._inifile):
                 warnings.warn(
-                    "The inifile is missing for unit definitions. "
-                    f"Specify it to the keyword argument 'inifile' (default: {self._default_inifile}), "
-                    "or make sure that all fields are in code units!",
+                    f"Could not find inifile ({self._inifile}). "
+                    "Inferred code units might be inaccurate\n"
+                    "To silence this warning, specify the inifile keyword argument to yt.load, "
+                    "as a path relative to the data file's parent directory "
```

Comment:
I think an absolute path is also acceptable here, right? Because `self._get_meta_file()` is able to accept an absolute path.

Reply:
good catch, I forgot about that !

Reply:
(updated)